vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -2686,6 +2694,17 @@ public void onClick(View v) {
         if (null != mUseMatrixAppsMenuItem) {
             mUseMatrixAppsMenuItem.setVisible(TextUtils.isEmpty(mEventId) && (null == sRoomPreviewData) && PreferencesManager.useMatrixApps(this));
         }
+
+        if (NotificationAreaUtils.onlyErrors(this)) {
+            int view = R.drawable.error == iconId ? View.VISIBLE : View.GONE;
+            mNotificationsArea.setVisibility(view);
+            findViewById(R.id.room_notification_separator).setVisibility(view);
```

Comment:
please bind this view with @BindView, to avoid to find it with findViewById each time you want to use it